tcms/core/tests/test_middleware.py

```diff
@@ -0,0 +1,19 @@
+import os
+import unittest
+from django.test import TransactionTestCase
+
+
+@unittest.skipUnless(
+    os.getenv('TEST_CHECK_UNAPPLIED_MIGRATIONS_MIDDLEWARE'),
+    'CheckUnappliedMigrationsMiddleware testing not enabled')
+class TestCheckUnappliedMigrationsMiddleware(TransactionTestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.unapplied_migration_message = "unapplied migration(s). See "\
+            "<a href=\"https://kiwitcms.readthedocs.io/en/latest/"\
+            "installing_docker.html#initial-configuration-of-running-"\
+            "container\">documentation</a>"
```

Comment:
Now that there is only 1 scenario you can move the expected message directly in the test method. We don't need extra setUp methods